lib/passports.js

```diff
@@ -105,14 +105,20 @@ module.exports = {
                 passport.tokens = JSON.stringify(query.tokens)
               }
 
-              // Save any updates to the Passport before moving on
-              app.services.FootprintService.update('Passport', passport.id, passport) .then(passport => {
-                app.services.FootprintService.find('user', passport[0].user.id)
-                  .then(user => {
-                    next(null, user)
-                  })
-                  .catch(next)
-              }).catch(next)
+              // I save any update to the Passport before moving on and
+              // I read the associated user instance
+              return Promise.all([
+                app.services.FootprintService.findAssociation("Passport", passport.id, "User"),
+                app.services.FootprintService.update("Passport", passport.id, passport)
```

Comment:
`.toJSON` works great and I thinks that remove the needs of the various `passport.tokens = JSON.stringify(query.tokens)` around in the code.
But for now I will change only this row, I'd like to have some good coverage before starting to refactor the trailpack